python/src/aiconfig/editor/client/src/shared/prompt_schemas/OpenAIVisionParserPromptSchema.ts

```diff
@@ -0,0 +1,124 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const OpenAIVisionParserPromptSchema: PromptSchema = {
+  input: {
+    type: "object",
+    required: ["data"],
+    properties: {
+      data: {
+        type: "string",
+      },
+      attachments: {
+        type: "array",
+        items: {
+          type: "attachment",
+          required: ["data"],
+          mime_types: ["image/*"],
+          properties: {
+            data: {
+              type: "string",
+            },
+          },
+        },
+      },
+    },
+  },
+  model_settings: {
+    type: "object",
+    properties: {
+      model: {
+        type: "string",
+      },
```

Comment:
Should we also add this as required field? (I know it doesn't do anything but having UI would still be helpful)

Reply:
IMO it would be weird to have as required but then actually not be required (because you can leave it empty and it works with the registered model name)